library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
 
-            *** Number-of-periodic-discharge-phases {suggestedTag=Property-not-possible-to-determine}
-                **** 1-periodic-discharge-phase
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** 2-periodic-discharge-phases
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** 3-periodic-discharge-phases
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Greater-than-3-periodic-discharge-phases
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+            *** RPP-sharpness <nowiki> {suggestedTag=Property-not-possible-to-determine} [Sharpness (for PDs and SW). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** RPP-spiky <nowiki> [Spiky (<70 ms, measured at the baseline). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** RPP-sharp <nowiki> [Sharp (70–200 ms). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** RPP-sharply-contoured <nowiki> [Sharply contoured. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** RPP-blunt <nowiki> [Blunt. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
 
-            *** Periodic-discharge-triphasic-morphology <nowiki> {suggestedTag=Property-not-possible-to-determine, suggestedTag=Property-exists,suggestedTag=Property-absence} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+            *** Number-of-RPP-phases <nowiki> {suggestedTag=Property-not-possible-to-determine, suggestedTag=Greater-than}[Number of phases (for PDs and SW): 1, 2 or 3. (Source: Beniczky ea 2017, Table 8.)] </nowiki>
+                **** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
 
-            *** Periodic-discharge-absolute-amplitude {suggestedTag=Property-not-possible-to-determine}
-                **** Periodic-discharge-absolute-amplitude-very-low <nowiki>[Lower than 20 microV.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Low-periodic-discharge-absolute-amplitude <nowiki>[20 to 49 microV.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Medium-periodic-discharge-absolute-amplitude <nowiki>[50 to 199 microV.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** High-periodic-discharge-absolute-amplitude <nowiki>[Greater than 200 microV.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+            *** RPP-triphasic-morphology <nowiki> [Triphasic morphology (for PDs and SW). (Source: Beniczky ea 2017, Table 8.)] </nowiki>
```

Comment:
I couldn't find any info on this by googling.

Reply:
Renamed Triphasic-morphology according to paper and added description: wave with three distinct phases.